browser.js

```diff
@@ -64,11 +64,31 @@ function init() {
   backButton();
 }
 
+function setDarkMode() {
+	document.documentElement.classList.toggle('dark-mode', config.get('darkMode'));
+    if (config.get('darkMode')) {
```

Comment:
BrowserWindow instances have a `ready-to-show` event that I think is implemented for this purpose: http://electron.atom.io/docs/api/browser-window/#using-ready-to-show-event